Dockerfile.dev

```diff
@@ -0,0 +1,11 @@
+FROM node:15
```

Comment:
change file name to `Dockerfile.development` (it's just because I have used this name in other projects)